src/py42/exceptions.py

```diff
@@ -337,6 +337,33 @@ def __init__(self, exception):
         super().__init__(exception, msg)
 
 
+class Py42TrustedActivityInvalidChangeError(Py42BadRequestError):
+    """An error raised when an invalid change is being made to a trusted activity."""
+
+    def __init__(self, exception):
+        msg = "Invalid change to trusted activity. Trusted activity type cannot be changed."
+        super().__init__(exception, msg)
+
+
+class Py42TrustedActivityConflictError(Py42HTTPError):
+    """An error raised when theres a conflict with a trusted activity domain URL."""
+
+    def __init__(self, exception, value):
+        msg = (
+            f"Duplicate URL or workspace name, '{value}' already exists on your trusted list.  "
+            "Please enter a unique value"
+        )
+        super().__init__(exception, msg)
+
+
+class Py42TrustedActivityInvalidCharacterError(Py42BadRequestError):
+    """An error raised when an invalid change is being made to a trusted activity."""
+
+    def __init__(self, exception):
+        msg = "Domain name cannot include @"
```

Comment:
Let's make this more generic in case the service decides that there are other invalid characters besides just `@`

Reply:
Changed this to `"Invalid character in domain or slack workspace name"`